### ISSUE TITLE
Multiple light source in a scene

### ISSUE THREAD

**Issue by Entongsu** [facebookresearch/pytorch3d#1171]
Hello,
 
Thank you for sharing this great project. 

I have a question about the lighting setting. I wonder whether pytorch3d supports Multiple light source rendering in the same scene. Thank you!

**Comment by patricklabatut** [comment]
No, not at this time, although it is some that has been discussed.

What would be your use case?
